File: {{cookiecutter.repo_name}}/src/R/workflow.r
source('data_collection.R')
source('preprocessing.R')
source('modeling.R')
source('utils.R')
source('visualization.R')

run_workflow <- function() {
  run_data_collection()
  run_preprocessing()
  run_modeling()
  run_utils()
  run_visualization()
}

run_workflow()

File: {{cookiecutter.repo_name}}/src/python/workflow.py

# Import the step scripts
import data_collection
import preprocessing
import modeling
import utils
import visualization

def run_workflow():
    data_collection.run_data_collection()
    preprocessing.run_preprocessing()
    modeling.run_modeling()
    utils.run_utils()
    visualization.run_visualization()

if __name__ == "__main__":
    run_workflow()